packages/video_player/video_player/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java

```diff
@@ -96,6 +97,18 @@
     setupVideoPlayer(eventChannel, textureEntry);
```

Comment:
Why isn't this needed in the other constructor? It seems like it should be.

This is why having two unrelated constructor paths tends to be problematic, and why I'd rather see this constructor call the other constructor.

Reply:
This new constructor is only needed for creating a mock VideoPlayer in tests. We don't need any additional set up, but these private final fields are required to be set.  (See tests in VideoPlayerTest.java)

Added a docstring about this constructor being used for mocks.